lob-api-public.yml

```diff
@@ -2514,6 +2528,15 @@ paths:
   /campaigns/{cmp_id}/send:
     $ref: resources/campaigns/send.yml
 
+  /buckslips:
+    $ref: resources/buckslips/buckslips.yml
+
+  /buckslips/{buckslip_id}:
+    $ref: resources/buckslips/buckslip.yml
+
+  /buckslips/{buckslip_id}/orders:
+    $ref: resources/buckslips/buckslip_orders/buckslip_orders.yml
+
```

Comment:
If you could just move these above the `/bulk/us_verifications` endpoint to maintain the alphabetical ordering, that'd be great ^^

Reply:
LTGM otherwise!